SPECS/trousers/trousers.spec

```diff
@@ -1,33 +1,34 @@
-Summary:      TCG Software Stack (TSS)
-Name:         trousers
-Version:      0.3.14
-Release:      6%{?dist}
-License:      BSD-3-Clause
-URL:          https://sourceforge.net/projects/trousers/
-Group:        System Environment/Security
-Vendor:       Microsoft Corporation
-Distribution: Mariner
-Source0:      https://sourceforge.net/projects/%{name}/files/%{name}/%{version}/%{name}-%{version}.tar.gz
-
-# CVE-2020-24330.patch fixes both CVE-2020-24330 and CVE-2020-24331.
-Patch0:       CVE-2020-24330.patch
-Patch1:       CVE-2020-24331.nopatch
-
-Requires:     libtspi = %{version}-%{release}
+Summary:        TCG Software Stack (TSS)
+Name:           trousers
+Version:        0.3.14
+Release:        7%{?dist}
+License:        BSD-3-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://sourceforge.net/projects/trousers
+Source0:        %{url}/files/%{name}/%{version}/%{name}-%{version}.tar.gz
+# CVE-2020-24330.patch also fixes CVE-2020-24331 and CVE-2020-24332.
+Patch0:         CVE-2020-24330.patch
+Patch1:         CVE-2020-24331.nopatch
+Patch2:         CVE-2020-24332.nopatch
```

Comment:
While you're here, could you change %setup to %autosetup?

Reply:
Fixed in latest commit, thanks!